docs/services/motion.md

```diff
@@ -147,7 +147,7 @@ This can be any pose, from the perspective of any component whose location is kn
 It will be converted into the frame of the component named in `component_name` when passed to `MoveToPosition`.
 
 **`world_state`**: This data structure is structured identically to what is described above in "Move", and is intended to behave the same.
```

Comment:
[major] L149 here indicates they are intended to behave the same, but in the implementation this is no longer true. `MoveSingleComponent` wraps the arm's `MoveToPosition` call, so the obstacle avoidance and collision checking features normally associated with passing in WorldState information are no longer available through `MoveSingleComponent`.

Actually, there might be some value in specifying that the Motion `Move` method is the ONLY way to have collision checking and obstacle avoidance given the recent changes.

Reply:
So does it not actually take a world_state parameter? or does it take it and not use it?

Reply:
`MoveSingleComponent` still takes in a `WorldState` (see https://github.com/viamrobotics/rdk/blob/main/services/motion/builtin/builtin.go#L143) but only for the supplemental transforms that might be needed to plan arm motion with the Frame System (L163 of that same file)

Reply:
And I say "plan" but its really just to figure out if there's any math to be done for poses with respect to the arm and goal frames.

Reply:
ah okay ty!